.github/workflows/ci.yml

```diff
@@ -10,17 +10,17 @@ name: Continuous Integration
 
 jobs:
   build:
-    runs-on: "ubuntu-20.04"
+    runs-on: "ubuntu-22.04"
```

Comment:
Would it make more sense to use `ubuntu-latest`?

Reply:
Good idea.